src/winrpath.h

```diff
@@ -183,6 +184,10 @@ class CoffParser {
     void ParseSecondLinkerMember(coff_member *member);
     void ReportLongImportMember(long_import_member *li);
     void ReportShortImportMember(short_import_member *si);
+    void NormalizeLinkerMember(const std::string &name, const int &base_offset, const int &offset, const char * strings, const DWORD symbols);
+    void NormalizeSectionNames(const std::string &name, char* section, const DWORD &section_data_start_offset, int data_size);
+    void NormalizeStringTable(PIMAGE_SYMBOL symbol, const char* string_table, const int &relative_offset, const int &size_of_string_table, const std::string &name);
```

Comment:
I don't see this function in the code now, and I don't see it added in this PR.

Likewise for GetSectionDataSize

Reply:
Ah, nice catch, thanks. 
Cruft I forgot to pull out after I pulled the larger code reorganization changed into another branch, must have had a sloppy commit in there.